packages/web-app/src/routes/questionnaires-for-review/check-and-confirm.spec.js

```diff
@@ -0,0 +1,17 @@
+const { get } = require('../../../test/routes/router-mock');
+const checkAndConfirmController = require('../../controllers/questionnaires-for-review/check-and-confirm');
+
+describe('routes/check-and-confirm', () => {
```

Comment:
Could the description use the full path, for example `routes/questionnaires-for-review/check-and-confirm`?